### PR TITLE
Add Fujitsu anywAIR

### PR DESCRIPTION
**Pull request by Bre77**
## Proposed change

Adds documentation for the Fujitsu Anywair virtual integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/102978
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/4828
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/fujitsu_anywair.markdown [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: Fujitsu anywAIR
+description: Instructions on how to integrate Fujitsu anywAIR controller into Home Assistant.
+ha_category:
+  - Climate
+  - Cover
+  - Light
+  - Sensor
+  - Update
+ha_release: 0.117
```
Comment (joostlek) [inline source/_integrations/fujitsu_anywair.markdown:10]:
```suggestion
ha_release: 2024.1
```

File: source/_integrations/fujitsu_anywair.markdown [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+title: Fujitsu anywAIR
+description: Instructions on how to integrate Fujitsu anywAIR controller into Home Assistant.
+ha_category:
+  - Climate
+  - Cover
+  - Light
+  - Sensor
+  - Update
+ha_release: 2024.1
```
Comment (frenck) [inline source/_integrations/fujitsu_anywair.markdown]: This cannot be correct. The virtual integration always need to match the version of introduction of the parent integration.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/fujitsu_anywair.markdown (hunk 2 above)
Comment (Bre77) [reply source/_integrations/fujitsu_anywair.markdown]: I got a review suggesting I change it to this, I'll try to revert that commit.

File: source/_integrations/fujitsu_anywair.markdown (hunk 1 above)
Comment (frenck) [reply source/_integrations/fujitsu_anywair.markdown:10]: That ain't correct Joost :)

File: source/_integrations/fujitsu_anywair.markdown (hunk 2 above)
Comment (Bre77) [reply source/_integrations/fujitsu_anywair.markdown]: Done

**Review by frenck — APPROVED** [review]
Thanks, @Bre77 👍 

../Frenck

**Pull request merged**